docs/clean_up.md

```diff
@@ -0,0 +1,69 @@
+# Clean up
+
+## Before you begin
+
+1. Set your PROJECT_ID environment variable:
+
+    ```bash
+    export PROJECT_ID=<YOUR_PROJECT_ID>
+    ```
+
+## Deleting Cloud Run deployment resources
+
+1. Delete the Cloud Run service deployed:
+
+    ```bash
+    gcloud run services delete retrieval-service
+    ```
+
+1. Delete service account:
+
+    ```bash
+    gcloud iam service-accounts delete \
+        retrieval-identity@$PROJECT_ID.iam.gserviceaccount.com
+    ```
+
+## Delete AlloyDB resources
+
+1. Set environment variables:
+
+    ```bash
+    export VM_INSTANCE=alloydb-proxy-vm
+    export CLUSTER=my-alloydb-cluster
+    export REGION=us-central1
+    export RANGE_NAME=my-allocated-range-default
+    ```
+
+1. Delete Compute Engine VM:
+
+    ```bash
+    gcloud compute instances delete $VM_INSTANCE
+    ```
+
+1. Delete AlloyDB cluster that contains instances:
+
+    ```bash
+    gcloud alloydb clusters delete $CLUSTER \
+        --force \
+        --region=$REGION \
+        --project=$PROJECT_ID
+    ```
+
+1. Delete an allocated IP address range:
+
+    ```bash
+    gcloud compute addresses delete $RANGE_NAME \
+        --global
+    ```
+
+## Delete Cloud Firestore resources
+
+1. Go to your [project settings][firebase-settings] in the Firebase console.
+
+1. At the bottom of the General settings page, click Delete project.
+
+1. Select each checkbox to acknowledge the effects of deleting your project.
+
+1. Click Delete project.
```

Comment:
It looks like we cannot delete the default database? But we could delete a default database https://firebase.google.com/docs/firestore/manage-databases#delete_a_database. I believe that if you don't use the firestore db then you shouldn't be charged anything. We should make it clear that delete the project is more extreme than a clean up.

Reply:
I did delete project since the firestore (docs/datastore/firestore.md) instructions asked user to create a project.

Reply:
Yes, should I just remove the delete project instruction overall and replace with deleting the database? (Though our set up firestore instructions did ask user to create a new project). I was viewing clean up instructions as cleaning up things that we have user create when following our instruction.

Reply:
My 2c is that the project requirement should be consistent. It's probably a "before you begin thing" and so doesn't need to be part of the clean up instructions.